i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md

```diff
@@ -2,34 +2,139 @@
 id: hardware-servers-lenovo-xcc-snmp
 title: Lenovo XCC SNMP
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
 
-### Centreon Plugin
+## Contenu du Pack
 
-Install this plugin on each needed poller:
+### Modèles
 
-``` shell
+Le Plugin Pack Centreon **Lenovo XCC SNMP** apporte un modèle d'hôte :
+
+* HW-Server-Lenovo-Xcc-SNMP-custom
+
+Il apporte le modèle de service suivant :
+
+| Alias           | Modèle de service                  | Description                    | Défaut |
+|:----------------|:-----------------------------------|:-------------------------------|:-------|
+| Hardware-Global | HW-Lenovo-Xcc-Hardware-Global-SNMP | Contrôle l'ensemble des sondes | X      |
+
+### Métriques & statuts collectés
+
+<Tabs groupId="sync">
+<TabItem value="Hardware-Global" label="Hardware-Global">
+
+Could not retrive metrics
```

Comment:
You have to manually add metrics when script cannot figure out which ones it should add